### ISSUE TITLE
Avoid the creation of double statements

### ISSUE THREAD

**Issue by emilieagreelist** [hectorperez/agreelist#35]
Maybe by suggesting that the same or similar statements already exist?

**Comment by hectorperez** [comment]
Yeah, we'll have to do something like that. Maybe as part of a full rewriting of the new opinion form in different steps
